store.go

```diff
@@ -46,7 +46,7 @@ func setStoreFormatFromData(data []byte) {
 		StoreFormat = YAML
 		return
 	}
-	log.Fatalf("Unable to determine format from content")
+	log.Printf("Unable to determine format from content")
```

Comment:
That sounds good, I'll try it out and test out autoadd'ing resources in both formats / replicating #157 issues.